test/org/traccar/protocol/Tk103ProtocolEncoderTest.java

```diff
@@ -2,11 +2,71 @@
 
 import org.junit.Assert;
 import org.junit.Test;
+import org.traccar.Context;
 import org.traccar.ProtocolTest;
+import org.traccar.database.IdentityManager;
 import org.traccar.model.Command;
+import org.traccar.model.Device;
+import org.traccar.model.Position;
 
 public class Tk103ProtocolEncoderTest extends ProtocolTest {
 
+    private static IdentityManager t580wManager = new IdentityManager() {
```

Comment:
I don't think this is necessary. There must be a better way.

Reply:
as you suggested above, i will implement exactly the same approach as it was done in GT06.